components/timeline/doc/index.en-US.md

```diff
@@ -44,7 +44,7 @@ Node of timeline
 
 | Property | Description | Type | Default |
 | -------- | ----------- | ---- | ------- |
-| `[nzColor]` | Set the circle's color to `'blue' \| 'red' \| 'green' \| 'gray'` or other custom colors (css color) | `string` | `blue` |
+| `[nzColor]` | Set the circle's color to `'blue' \| 'red' \| 'green' \| 'gray'`| `string` | `blue` |
 | `[nzDot]` | Customize timeline dot | `string \| TemplateRef<void>` | - |
 | `[nzPosition]` | Customize position, only works when `nzMode` is `custom` | `'left' \| 'right'` | - |
-| `[NzLabel]` | Set the label |  `string \| TemplateRef<void>` | - |
\ No newline at end of file
+| `[NzLabel]` | Set the label |  `string \| TemplateRef<void>` | - |
```

Comment:
```suggestion
| `[nzLabel]` | Set the label |  `string \| TemplateRef<void>` | - |
```
Why this line changed? EOF? Whatever, how about correcting the camelcase naming 😊